src/kibana/components/notify/_notifier.js

```diff
@@ -87,6 +87,14 @@ define(function (require) {
     return rtn;
   }
 
+  // browsers format Error.stack differently; always include message
+  function formatStack(err) {
+    if (err.stack && !~err.stack.indexOf(err.message)) {
+      return 'Error: ' + err.message + '\n' + err.stack;
+    }
+    return err.stack;
```

Comment:
There are three cases being dealt with:
* `err instanceof Error`
  * with message in the stack (Chrome)
  * without message in the stack (Safari, Firefox)
* `typeof err === 'string'`

I think the only case where `stack` would be empty is when err is a string (otherwise on a weird browser where Error doesn't have a stack). The message should already be included in the notify (from `formatMsg`).